tests/integration/test_storage.sh

```diff
@@ -35,10 +35,13 @@ setup () {
     # Perform setup routines here.
     copy /etc/sysconfig/docker-storage-setup /etc/sysconfig/docker-storage-setup.atomic-tests-backup
     TEST_DEV_1=/dev/vdb
```

Comment:
What I mean is, this should be defined externally rather than hardcoded. It's not guaranteed to be `/dev/vdb` on every machine where this test is going to run.